src/pages/Graph/SummaryPanelNodeTraces.tsx

```diff
@@ -174,7 +174,7 @@ class SummaryPanelNodeTraces extends React.Component<Props, State> {
           </SimpleList>
         )}
         <Button style={summaryFont} onClick={() => history.push(tracesDetailsURL)}>
-          Go to Tracing
+          View App Traces
```

Comment:
The verb 'Show' is more consistent with the graph context and kebab menu options.  So I think `Show Traces`, `Show App Traces` or `Show Application Traces` would be better.